kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/index.vue

```diff
@@ -141,6 +238,18 @@
         this.goBack = goBack;
       },
     },
+    $trs: {
```

Comment:
Let's put these in the common string file for this feature, as well as the other component strings that you're introducing in this PR. I guess I have not been very diligent about this on 0.18 with every PR, but I think some of these we may end up reusing in Quiz updates, and also - having them in one place for tracking, even if it seems like they are only used within this component, will be helpful when we get to string freeze time.

Reply:
by that I mean `searchAndFilterStrings.js`

Reply:
These strings are very generic actually, not just for this feature. We already had them in the [enhancedQuizManagementStrings](https://github.com/AlexVelezLl/kolibri/blob/e9528766776817190b1e579f58e2db056316da6a/packages/kolibri-common/strings/enhancedQuizManagementStrings.js#L145), I didnt use those here because it felt a little weird importing that file in this context. I think we can move this to the `commonCoachStrings` instead?

Reply:
yes, let's put them in commonCoach then if they will be reused across many places - that sounds fine. If there were other strings that are part of this PR (I saw some in a few different files), we can put them either in searchAndFilterStrings.js or in commonCoach, depending on how widely they are used. thank you!

Reply:
Alright, done! And yes, I had already moved the other strings in this PR too :)